Essentials/src/net/ess3/api/events/TPARequestEvent.java

```diff
@@ -0,0 +1,55 @@
+package net.ess3.api.events;
+
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+import com.earth2me.essentials.CommandSource;
+
+import net.ess3.api.IUser;
+
+
+public class TPARequestEvent extends Event implements Cancellable {
+	private static final HandlerList handlers = new HandlerList();
+	private CommandSource requester;
+	private IUser requested;
+	private boolean cancelled = false, requestToTPAHere = false;
+
+	public TPARequestEvent(CommandSource requester, IUser requested, boolean tpaHere) {
+		super();
+		this.requester = requester;
+		this.requested = requested;
```

Comment:
I'd suggest renaming this to `target` as it seems more in line with other references to the person receiving the request.

Reply:
Done!